src/Terminus.php

```diff
@@ -314,6 +314,7 @@ private function addBuiltInCommandsAndHooks()
         // List of all hooks and commands. Update via 'composer update-class-lists'
         $this->commands = [
             'Pantheon\\Terminus\\Hooks\\Authorizer',
+            'Pantheon\\Terminus\\Hooks\\CommandTracker',
```

Comment:
Is this supposed to be here? Was it removed in the previous PR, or is it new?

Reply:
This is from my previous PR; I assumed it was accidentally removed in a previous PR and readded here

Reply:
Ohhh! I actually never added it here but it was working. I don't remember right now if I had reasons to do that but I know for sure it's working without this line here so now I'm not so sure about this line...

Reply:
This just makes sure it gets included when looking for commands.